generators/generator-bot-adaptive/generators/app/templates/dotnet/functions/Triggers/StaticFilesTrigger.cs

```diff
@@ -0,0 +1,66 @@
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+
+namespace <%= botName %>.Triggers
+{
+    public class StaticFilesTrigger
+    {
+        private const string StaticFilesDirectory = "www";
```

Comment:
Should this be "wwwroot" instead so it's inline with the static file version counterpart in the web app version?

Reply:
I guess the flip side is on Azure Function the web app seems to be on wwwroot already so it's like wwwroot/wwwroot.

Reply:
Yeah, it's a little weird for sure. Looking at other implementations in places, this seems to be the way it's been handled. At the end of the day, customers can easily change it to something a little more fitting should they wish.